examples/brats18/configs/brats18_central/config/config_fed_client.json

```diff
@@ -0,0 +1,33 @@
+{
+  "format_version": 2,
+
+  "handlers": [
+  ],
+  "executors": [
+    {
+      "tasks": ["train", "submit_model"],
+      "executor": {
+        "path": "pt.trainers.brats_trainer.BratsTrainer",
+        "args": {
+          "aggregation_epochs":1,
+          "train_config_filename": "config/config_train.json"
+        }
+      }
+    },
+    {
+      "tasks": ["validate"],
```

Comment:
Is there a reason to use different executor for "validate"? Can you use BratsTrainer to do all tasks?